### DISCUSSION TITLE
C++ MFC Unit Test with Native Unit Test case

### DISCUSSION THREAD

**Discussion by MOHITHSHIVU03** [community/community#56083] [Programming Help]
### Select Topic Area

Question

### Body

Hi All,

I have created the MFC DialogBox for the login page for to write Simple Native Unit Test case for that.
but for this, I have configured the MFC application I did like give the MFC application header file path in C++> General, and also I linked the proper header file with that in C++ > lINKER > input also. still, I am facing the issue while adding the MFCDlg.h and MFCApp.h header files. if anyone knows more about this. Please give me the solution for this.

if I include MFCApp.h I am getting errors like 

Severity	Code	Description	Project	File	Line	Suppression State
Error (active)	E0035	#error directive: "include 'pch.h' before including this file for PCH"	LoginWindowSampleUnitTest	C:\Users\mohith.n\source\repos\MFCLoginWindow\MFCLoginWindow\MFCLoginWindow.h	6	

If I include MFCDlg.h. I am getting errors like

Severity	Code	Description	Project	File	Line	Suppression State
Error	C2504	'CDialogEx': base class undefined	LoginWindowSampleUnitTest	c:\users\mohith.n\source\repos\mfcloginwindow\mfcloginwindow\mfcloginwindowdlg.h	10	

. Please anyone knows how to set up the Native Unit Test case in the MFC application Please help out for me give the setup solution.

**Comment by chanachai004** [comment]
https://data.hud.gov/housing_counseling.html
